src/app/shared/services/fill-filter.service.ts

```diff
@@ -0,0 +1,62 @@
+import { HttpClient, HttpHeaders } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { from, Observable } from "rxjs";
+import {client_id, api_token} from "../../../assets/creds"
+
+@Injectable({
+    providedIn:'root'
+})
+export class FillFilterService {
+
+  constructor(private http: HttpClient){   }
+
+  getRatings(): Observable<[{id: number, rating: number}]> {
+    const headerDict = {
```

Comment:
You can set this variable inside this service and use it where it's needed, but you will have more services, so HTTP Interceptor will be a better solution